### PR TITLE
feat: test dd31 enabled reflect

### PR DESCRIPTION
**Pull request by grgbkr**

### CODE REVIEW

File: worker/index.ts [hunk 1]
```diff
@@ -50,7 +50,7 @@ const { worker, RoomDO, AuthDO } = createReflectServer({
     await clearCursorAndSelectionState(write, { id: write.clientID });
   },
   getLogSinks,
-  getLogLevel: () => "info",
+  getLogLevel: () => "debug",
```
Comment (phritz) [inline worker/index.ts:53]: Just to confirm, this doesn't negatively affect perf? I think I recall that monday perhaps doesn't enable debug logging for perf reasons? @aboodman would know.

File: package.json [hunk 2]
```diff
@@ -14,8 +14,8 @@
     "@cloudflare/workers-types": "^3.11.0",
     "@datadog/browser-logs": "^4.11.3",
     "@rocicorp/logger": "^2.2.0",
-    "@rocicorp/reflect": "^0.10.0",
-    "@rocicorp/reflect-server": "^0.18.0",
+    "@rocicorp/reflect": "file:rocicorp-reflect-0.11.0.tgz",
```
Comment (phritz) [inline package.json:17]: I'm unclear on why we switch to a file?

Comment (aboodman) [reply package.json:17]: It's just to enable testing on a vercel preview / branch, we'd not check this in.

File: worker/index.ts (hunk 1 above)
Comment (aboodman) [reply worker/index.ts:53]: Monday definitely complained about this. I'm not sure about replidraw.

Comment (grgbkr) [reply worker/index.ts:53]: I think it does, I just have it turned up to debug while i am testing that the dd31 changes work.

**Comment by aboodman** [comment]
Did this run for you @grgbkr -- it fails for me with:

Screen Shot 2022-12-16 at 10 31 33 PM

**Pull request closed without merging**